packages/api/amplify_api/example/integration_test/graphql/api_key_test.dart

```diff
@@ -116,6 +116,27 @@ void main({bool useExistingTestUser = false}) {
 
             expect(blogFromEvent?.name, equals(name));
           });
+
+          testWidgets('should parse errors within a web socket data message',
+              (WidgetTester tester) async {
+            final name =
+                'Integration Test Blog - subscription create ${uuid()}';
+            const error =
+                'Cannot return null for non-nullable type: \'AWSDateTime\' within parent \'Blog\' (/onCreateBlog/createdAt)';
```

Comment:
Rules for strings are:
- Use single quotes by default
- Use double-quotes when the string contains single-quotes
- Prefer raw strings `r` instead of escapes (`\`) 
- Wrap strings if they break the 80-char limit

Any two touching strings in Dart are concatenated, e.g. `'Hello, ' 'world'`, so you can do:

```suggestion
                "Cannot return null for non-nullable type: 'AWSDateTime' within "
                "parent 'Blog' (/onCreateBlog/createdAt)";
```